src/frontend/demo/package.json

```
@@ -5,6 +5,7 @@
   "private": false,
   "dependencies": {
     "@jitsi-magnify/core": "0.1.0",
+    "axios": "^0.27.2",
```

We must pin dependencies to ensure reproducibility. We can add renovate to keep our dependencies upgraded with automatic Pull Requests:
```suggestion
    "axios": "0.27.2",
```